Video to PDF/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Tkinter
+FPDF
+Speech Recognition
+MoviePy
```

Comment:
mention them in the format so that one can install them using:
```
pip install -r requirements.txt
```
Check this file: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt